docs/how-tos/validate-nexus-file.md

```diff
@@ -0,0 +1,802 @@
+
+Keep in mind, that the output provides quite some information. This is useful for software development, but may be a bit too much for validation purposes.
+
+Similar features as the tables and messages provided from punx and cnxvalidate for pynxtools are [planned to be implemented in the future](<https://github.com/FAIRmat-NFDI/pynxtools/pull/333>).
+
+
+
+
+
+
+
+
+
+
+# 2. cnxvalidate
+
+This package is written in C. It is allows a command line evocation like:
+
+```
+nxvalidate -l appdefdir datafile
+```
+
+1. nxvalidate: calls the software function
+
+2. \-l appdefdir: points to the location of the NeXus definitions you want to use. This is a path to a folder called "defintions".
+
+3. datafile: This is the path to the .nxs file which should be checked.
+
+This output shows warnings like:
+
+```
+definition=NXoptical_spectroscopy.nxdl.xml message="Required attribute URL missing" nxdlPath=/NXentry/definition sev=error dataPath=/entry/definition dataFile=NXopt_minimal_example.nxs
+```
+
+and indicates the entry of the .nxs file, which is incorrect and what the respective problem is. It also points to the NeXus definition (.nxdl.xml file), in which this conflict was found.
+
+# 2.1. Installation (Linux only)
+
+This [did not work for me on windows](/installation_notes_nxvalidate.md) (The problem was the software cmake couldn't fine the libxml2 library. Though, if you solve this, this maybe work on windows).
+
+Therefore: Use linux.
+
+The installation process has to be build from source. This is eased significantly by using another software called: cmake.
+
+### Install cmake, github, hdf5 & xml2 library, etc:
+
+Open the terminal and install all parts required to install cnxvalidate via cmake:
+
+```
+sudo apt-get update
+sudo apt-get install git
+sudo apt-get install build-essential
+sudo add-apt-repository universe
+sudo apt-get install libhdf5-serial-dev
+sudo apt-get -y install pkg-config     
+sudo apt upgrade -y
+sudo apt-get -y install cmake
+sudo apt-get install libxml2-dev
+```
+
+### Directoty location
+
+create a folder named "nexusvalidation" via terminal or file manager.
+
+The folder is located at `/home/USER/nexusvalidation`
+
+"USER" is your user name. (You can get your username by the terminal command: `echo $USER`)
+
+In the terminal, this is indicated by `~/nexusvalidation` ( \~ = /home/USER)
+
+open the thermal and go into this directory by:
+
+```
+cd /home/USER/nexusvalidation
+```
+
+### Using GitHub
+
+Go to the [Github Repository of cnxvalidate:](<https://github.com/nexusformat/cnxvalidate>)
+
+Click on the green "<> Code" button.
+
+Click on "HTTPS".
+
+Copy the https link.
+
+![image.png](<./attachments/e28ec15bb537c9b7-image.png>)
+
+open the terminal and ensure you are in the `nexusvalidation` folder.
+
+clone the github repository (= download the files of the software).
```

Comment:
```suggestion
Open the terminal and ensure you are in the `nexusvalidation` folder.

Clone the github repository (= download the files of the software).
```
suggestion to make each line start either capizalized or not, but consistenly so (also below)